src/client/components/base/Button.tsx

```diff
@@ -71,7 +71,6 @@ const Button: React.FC<ButtonProps> = ({
     <button
       className={classes}
       onClick={(e) => {
-        type !== 'submit' && e.preventDefault();
```

Comment:
IDE warning that this line does nothing, since its not an if nor is the value stored in a variable